tools/pylib/boutdata/mms.py

```diff
@@ -312,7 +312,7 @@ def add(self, expr, name):
         self._extra[name] = expr
 
 
-    def write(self, nx, ny, output):
+    def write(self, nx, ny, MXG = 2, output):
```

Comment:
Can you change the def to this (likewise on L510):

    def write(self, nx, ny, output, MXG=2):

Arguments with defaults should go at the end, preferably. Also, there shouldn't be spaces around the equals in the argument definition.